app.js

```diff
@@ -13,6 +13,7 @@ const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
 app.use(logger(formatsLogger))
 app.use(cors())
 app.use(express.json())
+app.use(express.static("pablik"))
```

Comment:
```suggestion
app.use(express.static("public"))
```
папку теж перейменуйте